_topics/en/silent-payments.md

```diff
@@ -66,12 +66,29 @@ see_also:
 ## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
 ## Should be less than 500 characters
 excerpt: >
-  **Silent payments** are a type of payment that can be made to a
-  unique onchain address for every payment even though the receiver
-  provided the spender with a reusable (offchain) address.  This helps
-  improve privacy.
+  **Silent payments** unlocks the ability to receive private payments from
+  anyone on a single static address without requiring any interaction or
+  extra on-chain overhead. This helps improve privacy.
 
 ---
 
+Traditionally, a new address is generated whenever a user receives
+two or more payments. This is because when you receive several payments
+to the same Bitcoin address, others can reasonably assume that the same
+person received all of those payments even if the payments are later
+spent in separate transactions. This is also known as address reuse,
+and more can be found under [Output linking][topic output linking].
+
+With silent payments, a user can generate a silent payment address
+and make it publicly known. The sender then takes a public key from
+one of their chosen inputs for the payment, and uses it to derive a
+shared secret which is then used to tweak the silent payment address,
+before sending the payment to the tweaked address.
+
+The intended recipient detects the payment by scanning every transaction
+in the blockchain. This is the main downside, along with the lack of
+light client support, and the requirement to control your own input(s)
```

Comment:
If we define a "light client" as a client performing SPV, which is the traditional definition, then light clients can support SPs just fine.  For example, see https://twitter.com/josibake/status/1782818325367525449

You could just drop this clause or you could mention that they're not compatible with `[compact block filters][topic compact block filters]` and require upgrades to `[block explorers][topic block explorers]`.

Reply:
TIL! thank you for linking that, i will add a mention for the latter 2

Reply:
Slightly pedantic, but silent payments does work with compact block filters with the caveat that the client needs the 33 bytes per transaction before being able to use compact block filters.

For example, I've been working on a light client for silent payments that does use a modified electrum server to vend only the tweak data, and then the wallet uses `BIP158` for everything else: 

* to check if the generated `scriptPubKey` exists in a block
* to download the block if there is a match to get the outpoint data needed to spend the transaction
* to broadcast the transaction to peers when spending

This improves on electrum style wallets in that the electrum server learns nothing about the wallets transactions, but comes at the cost of needing more bandwidth than a traditionally electrum-backed wallet.